tests/core/notification/test_events_published.py

```diff
@@ -249,7 +251,7 @@ def test_job_events():
     consumer.start()
 
     # Create scenario
-    scenario = tp.create_scenario(sc_config)
+    scenario = _ScenarioManagerFactory._build_manager()._create(sc_config)
```

Comment:
I don't get why you replaced `scenario = tp.create_scenario(sc_config)` by 
`scenario = _ScenarioManagerFactory._build_manager()._create(sc_config)`.